lib/cdo/rake_utils.rb

```diff
@@ -169,6 +169,14 @@ def self.git_add(*args)
     system 'git', 'add', *args
   end
 
+  def self.git_user_name
+    `git config --get user.name`
+  end
+
+  def self.git_user_email
+    `git config --get user.email`
+  end
+
```

Comment:
This belongs to a different PR :P 

Making the base commit that other one will remove these files on the changes but it wouldn't run done :(